test/end-to-end/test/test_editRecipe.js

```diff
@@ -136,7 +147,10 @@ describe('Edit Recipe Page', function () {
           })
           .then(() => nightmare
             .wait(toastNotifPage.iconComplete)
-            .wait(toastNotifPage.labelStatus)
+            .wait((page) => {
+              const recipeName = document.querySelector(page.labelRecipeName).innerText;
+              return recipeName !== page.varEmptyName && recipeName.includes(page.varEmptyName);
+            }, toastNotifPage)
```

Comment:
Due to the page render is async, the label is rendered as ": Creating composition." without recipe name. When the api fetch get returned, the page will be rendered again with recipe name added. In this case we need to wait until a full notification like "automation: Creating composition." rendered.